### ISSUE TITLE
[Vue warn]: Do not use built-in or reserved HTML elements as component id: Text/Image

### ISSUE THREAD

**Issue by fr3ddy** [surveyjs/survey-library#2733]
### Are you requesting a feature, reporting a bug or asking a question?
Bug

### What is the current behavior?
Receiving warnings when using the package.
`[Vue warn]: Do not use built-in or reserved HTML elements as component id: Text`
and
`[Vue warn]: Do not use built-in or reserved HTML elements as component id: Image`
are displayed with every refresh of the page.

### What is the expected behavior?
No warnings are shown in the console when using surveyJs.

### How would you reproduce the current behavior (if this is a bug)?
Run attached project and see error appears, even without any survey used.

#### Provide the test code and the tested page URL (if applicable)

My package.json in the clean project:
```
{
  "name": "surveyjs",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint"
  },
  "dependencies": {
    "@vue/cli-service": "^4.5.12",
    "survey-vue": "^1.8.36",
    "vue": "^2.6.11"
  },
  "devDependencies": {
    "vue-template-compiler": "^2.6.11"
  }
}
```
Full project can be found 
[attached](https://github.com/surveyjs/survey-library/files/6183218/surveyjs.zip).

### Specify your

* browser: Chrome
* browser version: 89.0.4389.90
* surveyjs platform (angular or react or jquery or knockout or vue): vue
* surveyjs version: 1.8.36

**Comment by andrewtelnov** [comment]
@fr3ddy Yes, I saw it yesterday in my console when debug a vue app. We will work on the fix.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@fr3ddy I have fixed it by the commit above.

Thank you,
Andrew

**Comment by fr3ddy** [comment]
@andrewtelnov Thank you for the fast fix! Any plans on when v1.8.37 is released?
Regards,
Freddy

**Comment by andrewtelnov** [comment]
@fr3ddy You can always find information about new release and about old releases on [this page](https://surveyjs.io/WhatsNew).

Thank you,
Andrew
